File: CoreJava/vehical-interface-demo-app/src/com/techlab/vehicals/Bike.java
package com.techlab.vehicals;

import com.techlab.imoveable.IMoveable;

public class Bike implements IMoveable {

	@Override
	public void start() {
		System.out.println("Bike Starts....");
	}

	@Override
	public void stop() {
		System.out.println("Bike Stops....");
	}
}

File: C#Basics/Interface-App/Vehical-App/Bike.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Vehical_App
{
	class Bike : IMoveable
	{

		void IMoveable.start()
		{
			Console.WriteLine("Bike Starts....");
		}


		void IMoveable.stop()
		{
			Console.WriteLine("Bike Stops....");
		}

	}
}
